container/container.go

```diff
@@ -88,7 +87,7 @@ type Container struct {
 	MountPoints            map[string]*volume.MountPoint
 	HostConfig             *containertypes.HostConfig `json:"-"` // do not serialize the host config in the json, otherwise we'll make the container unportable
 	ExecCommands           *exec.Store                `json:"-"`
-	DependencyStore        agentexec.DependencyGetter `json:"-"`
+	DependencyStore        interface{}                `json:"-"`
```

Comment:
It will allocate a pointer, which is always nil. I don't see it defined anywhere else